Pipfile

```diff
@@ -0,0 +1,58 @@
+[[source]]
```

Comment:
General approach for setting up Pipfile is to set minimum versions but not fix any versions unless necessary.

Reply:
TO DO: [Document use of pipenv and whether we should use an alternative · Issue #749 · ekansa/open-context-py](https://github.com/ekansa/open-context-py/issues/749)

Reply:
OK!